lib/root.js

```diff
@@ -29,3 +29,46 @@ tap.mocha = require('./mocha.js')
 tap.mochaGlobals = tap.mocha.global
 tap.Test = Test
 tap.synonyms = require('./synonyms.js')
+
+// Handle sigterm signals, since this usually indicates a timeout
+process.on('SIGTERM', function onTerm () {
+  process.removeListener('SIGTERM', onTerm)
+  if (process.listeners('SIGTERM').length === 0) {
```

Comment:
What does this conditional do if tests set their own `SIGTERM` listener for some reason?